bin/xbps-query/show-info-files.c

```diff
@@ -214,13 +214,15 @@ show_pkg_files(xbps_dictionary_t filesd)
 			obj = xbps_array_get(array, x);
 			if (xbps_object_type(obj) != XBPS_TYPE_DICTIONARY)
 				continue;
-			xbps_dictionary_get_cstring_nocopy(obj, "file", &file);
-			printf("%s", file);
-			if (xbps_dictionary_get_cstring_nocopy(obj,
-			    "target", &file))
-				printf(" -> %s", file);
-
-			printf("\n");
+			if (long_listing)
+				fmt = "{mode!strmode} {user} {group} {size!humanize} {file}";
+			else
+				fmt = "{file}";
+			if (xbps_dictionary_get(obj, "target") != NULL)
+				fmt = xbps_xasprintf("%s -> {target}\n", fmt);
+			else
+				fmt = xbps_xasprintf("%s\n", fmt);
+			xbps_fmts_dictionary(fmt, obj, stdout);
```

Comment:
with this code, I'm getting
```
33184   0 /opt/foo/owned_by_nobody
33184   0 /opt/foo/owned_by_nogroup
34212   0 /opt/foo/sgid_nox_file
34285   0 /opt/foo/sgidfile
33188   0 /opt/foo/some_other_file
33256   0 /opt/foo/somefile
35236   0 /opt/foo/suid_nox_file
35309   0 /opt/foo/suidfile
41471   0 /opt/foo/symlink -> /opt/foo/foo
```
and
```
   29429040 /usr/bin/chezmoi
   11244 /usr/share/bash-completion/completions/chezmoi
   7074 /usr/share/fish/vendor_completions.d/chezmoi.fish
   1081 /usr/share/licenses/chezmoi/LICENSE
   7549 /usr/share/zsh/site-functions/_chezmoi
```
so it seems conversions aren't working. am i doing something wrong? did my implementation of tostrmode mess something up?